lib/csrf.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const csurf = require('csurf');
+
+module.exports = (linz) => csurf(linz.get('csrf options'));
```

Comment:
Can't this retrieve `linz` itself? It's not a pattern we've really used before...

Reply:
@smebberson Do you mean require linz in the function?
I tried just using `linz.get`, but I don't think linz is initialised at that point so we don't have the `get` function.

Reply:
Yeah, I mean using the requiring Linz. So you tried that and it didn't work?

Reply:
Yeah, the get function wasn't defined yet. Without restructuring the linz initialisation code, I don't think we can just require linz.

Reply:
If you move the `require('./lib/csrf');` in `linz.js` down a little until after Linz has initialised it will work. I think that's the lesser of the two evils at the moment and in more line with other Linz coding?